src/main/resources/assets/propulsive/lang/en_US.json

```diff
@@ -2,5 +2,6 @@
   "category.propulsive.keys": "Propulsive Keybinds",
   "key.propulsive.open": "Open Map Screen",
   "death.attack.star": "%s flew too close to the sun",
-  "death.attack.star.player": "%s flew too close to the sun whilst trying to escape %s"
+  "death.attack.star.player": "%s flew too close to the sun whilst trying to escape %s",
+  "itemGroup.propulsive.propulsive_group": "Propulsive Stuff"
```

Comment:
No need for "Stuff", just "Propulsive". Also change the last part of the ID to "propulsive" to match my suggestion in the item group code. Same changes apply in `en_GB.json` of course.